### ISSUE TITLE
Support reading from STDIN

### ISSUE THREAD

**Issue by onny** [Azure/azure-storage-azcopy#46]
Hey,
would be nice to transfer files with azcopy which are read by STDIN, like:
```
cat MYFILE | azcopy cp "https://myaccount.blob.core.windows.net/mycontainer/1file.txt?sastokenhere"
```
This is already implemented in azcopy v7.

Best regards,
Jonas

**Comment by zezha-msft** [comment]
Hi @onny, thanks for reaching out! This feature is already implemented in V10 as well, we just haven't documented it officially yet.

The upload scenario works great, but the download scenario currently have a bug because we recently started printing "Scanning..." when the copy operation starts, this string unfortunately gets piped into the destination file when downloading. I'll fix this as soon as possible and also add documentation for this feature.

**Comment by seguler** [comment]
This bug seems to be raised for the STDIN ( the upload scenario)... So there is no bug here.. Right ?

**Comment by zezha-msft** [comment]
@seguler yes you are right, I just wanted to use this issue for tracking purpose. I updated the tags accordingly. Thanks!

**Comment by zezha-msft** [comment]
Hi @onny, this was fixed in v10.0.1.

**Comment by ItalyPaleAle** [comment]
Hi @zezha-msft  how do you use STDIN/STDOUT with AzCopy? I'm looking for the same thing
